workspaces/local-cli/tsconfig.json

```diff
@@ -7,9 +7,8 @@
     "rootDir": "src",
     "strict": true,
     "target": "es2017",
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "esModuleInterop": true
```

Comment:
Was there any particular reason this was required?

Reply:
Yeah, basically for any CommonJS module that you try to `import` which used the `module.exports = {}` syntax instead of `exports.something = {}`, as far as I remember. With the latter `import * as` works, but not with the former. In this case, its as `url-join` that required it to be enabled.